### PR TITLE
added check for blank question

### PR DESCRIPTION
**Pull request by briri**
edge case where a test template had a blank question text was causing an error

### CODE REVIEW

File: app/views/shared/export/_plan.erb [hunk 1]
```diff
@@ -34,7 +34,7 @@
                   <% end %>
                 </ul>
               <% else %>
-                <p><%= raw question[:text][0].gsub(/<tr>(\s|<td>|<\/td>|&nbsp;)*(<\/tr>|<tr>)/,"") %></p>
+                <p><%= raw question[:text][0].gsub(/<tr>(\s|<td>|<\/td>|&nbsp;)*(<\/tr>|<tr>)/,"") if question[:text].present? && question[:text][0].present?  %></p>
```
Comment (jollopre) [inline app/views/shared/export/_plan.erb]: I don't get why are we baking new hash structures to represent phases, sections, questions. We are not only loosing the references defined in models but also we have to figure out keys and values for this newly structures.
